.gitignore

```diff
@@ -1,10 +1,11 @@
 # Ignoring Unity meta files since this is a code only plugin
 *.meta
-# Except for the sample projects and loading screen textures
+# Except for the sample projects and loading screen files
 # Make sure these are copied over in build.gradle's copyFiles function,
 # if they are required for tests.
 !GooglePlayInstant/Samples/**/*.meta
 !GooglePlayInstant/LoadingScreen/Textures/**/*.meta
+!GooglePlayInstant/LoadingScreen/Scripts/**/*cs.meta
```

Comment:
Slight preference to include an extra ".":
`!GooglePlayInstant/LoadingScreen/Scripts/**/*.cs.meta`